test/trento/hosts/host_test.exs

```diff
@@ -1899,6 +2014,110 @@ defmodule Trento.Hosts.HostTest do
       )
     end
 
+    test "should restore a deregistered host and trigger software updates discovery" do
```

Comment:
I miss a `should restore a deregistered host and trigger the software updates cleaned event` or similar, if the new coming discovery has fqdn nil, and it was set before

Reply:
same as here https://github.com/trento-project/web/pull/2507/files#r1560611137